src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -2699,10 +2736,10 @@ class CgroupUtils(object):
         except NameError:
             basestring = str
         # We want a view/iterator object, not something that creates copied
-        # lists. Unfortunately, iteritems/viewitems() no longer exists in Py3
+        # lists. iteritems/viewitems() no longer exists in Py3
         # since items() itself is now implemented as a view object.
-        dict_iter_object = (config_dict.items() if sys.version_info[0] >= 3
-                            else config_dict.iteritems())
+        dict_iter_object = (config_dict.iteritems() if PYTHON2
+                            else config_dict.items())
```

Comment:
Can't you use items() for both?  In python2 it is just more efficient to use iteritems()?

Reply:
No, you can't use items() for both.